### PR TITLE
Focus AR/VR Guide use case on AR

### PR DESCRIPTION
**Pull request by mmccool**
Christine Perey and I met and wanted to review the AR/VR Guide use case to capture better its relationship to WoT, e.g. displaying data from WoT devices and sensors.   These points were discussed previously via email with the original submitted.  

This PR should be discussed with the original submitter before being merged as some significant changes have been made.  In particular, the original description had many options in the original descriptions (AR or VR, headset or handheld display, etc) which made the description confusing and the requirements less clear. So we refocused the description on the "AR with a headset" use case which is the one that surfaces real-world geolocation information as well as some specific security concerns (which I have also added).  The other possibilities have been added to "variants".

In addition I added some "Requirements" to capture the discovery requirements, at least.  For example, AR needs geospatially-aware discovery and ideally over an FoV region.  There are also some complications in dealing with moving sources for which data may be old.  In these cases projected, "expected" positions need to be used (and the geospatial discovery needs to deal with a "cone of uncertainty").

***
<a href="https://pr-preview.s3.amazonaws.com/mmccool/wot-usecases/pull/94.html" title="Last updated on Feb 2, 2021, 4:08 PM UTC (49f9368)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-usecases/94/81bb852...mmccool:49f9368.html" title="Last updated on Feb 2, 2021, 4:08 PM UTC (49f9368)">Diff</a>

### CODE REVIEW

**Comment by mlagally** [comment]
UC call on 9.2.:
@rjksmith , can you please review and comment / update the MR as appropriate?

**Review by rjksmith — COMMENTED** [review]
The proposed changes look good.

1. I recommend that issues of geolocation security and privacy are handled in accordance with the recent [Responsible Use of Spatial Data](https://w3c.github.io/sdw/responsible-use/) guidance from [SDWIG](https://www.w3.org/2017/sdwig/).

1. I also suggest that @ashimura should be asked to review and comment to as he submitted a significant portion of this content.

Hope this helps.

File: index.html [hunk 1]
```diff
@@ -6119,32 +6123,39 @@ <h2>VR/AR Virtual Guide (for gaming, medical purposes, etc.)</h2>
...
         <dt>Expected Devices</dt>
         <dd>
 
           <ul>
-            <li>Display (possibly a VR Goggle) / speakers for output</li>
-            <li>3D camera / motion sensor for input (possibly microphone for speech recognition)</li>
-            <li>other various devices as output actuators and/or input sensors</li>
+            <li>Wearable, semi-transparent head-mounted display</li> 
+            <li>Headphones for speakers for audio output</li>
+            <li>Geopose and motion estimator (various technologies can be used)</li>
```
Comment (mlagally) [inline index.html:6142]: The term "Geopose" should be explained.

Comment (mmccool) [reply index.html:6142]: Could simply change that to "Location and orientation".  I would say just changing it to "pose" but not everyone knows what that means either ;)

**Comment by mmccool** [comment]
Can add SDW reference as well (and cite in security section)

**Pull request merged**